site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -838,7 +838,7 @@ public function showStatus($gradeable_id) {
      * Shows the list of submitters
      * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/grading/details")
      */
-    public function showDetails($gradeable_id, $sort = "id", $direction = "ASC") {
+    public function showDetails($gradeable_id, $direction = "ASC") {
```

Comment:
since you're modifying the line anyway, it might be good to go ahead and add type hints here for the parameters and return value.